src/scripting/natives/natives_commands.cpp

```diff
@@ -124,6 +124,23 @@ static void IssueClientCommand(ScriptContext& script_context)
     globals::engine->ClientCommand(CPlayerSlot(entity_index), command);
 }
 
+static const char* GetClientConVarValue(ScriptContext& script_context)
+{
+    auto entity_index = script_context.GetArgument<int>(0);
+    auto convarName = script_context.GetArgument<const char*>(1);
+
+    return globals::engine->GetClientConVarValue(CPlayerSlot(entity_index), convarName);
+}
+
+static void SetFakeClientConVarValue(ScriptContext& script_context)
+{
+    auto entity_index = script_context.GetArgument<int>(0);
+    auto convarName = script_context.GetArgument<const char*>(1);
+    auto convarValue = script_context.GetArgument<const char*>(1);
+
+    globals::engine->SetFakeClientConVarValue(CPlayerSlot(entity_index), convarName, convarValue);
+}
```

Comment:
This is probably copy pasta from my own bad code, but just to make this more obvious.

```suggestion
static const char* GetClientConVarValue(ScriptContext& script_context)
{
    auto playerSlot = script_context.GetArgument<int>(0);
    auto convarName = script_context.GetArgument<const char*>(1);

    return globals::engine->GetClientConVarValue(CPlayerSlot(playerSlot), convarName);
}

static void SetFakeClientConVarValue(ScriptContext& script_context)
{
    auto playerSlot = script_context.GetArgument<int>(0);
    auto convarName = script_context.GetArgument<const char*>(1);
    auto convarValue = script_context.GetArgument<const char*>(1);

    globals::engine->SetFakeClientConVarValue(CPlayerSlot(playerSlot), convarName, convarValue);
}
```